files/lib/preinit/95_keymap

```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 Nethesis
+
+KMAPDIR="/usr/share/keymaps"
+[ -n "$lang" ] && [ -s $KMAPDIR/"$lang".map.bin ] && cat $KMAPDIR/"$lang".map.bin | loadkmap
```

Comment:
Where is defined the `$lang` variable and what is the default value?

Reply:
Reading the description, I guess the variable comes from grub and the default is empty. Am I right?

Reply:
Right